docs/management_api.yml

```diff
@@ -61,6 +61,40 @@ paths:
                 request_id: "eed14d55-d996-42cd-8248-e806663810aa"
         500:
           $ref: '#/components/responses/InternalServerError'
+  /devices/{id}/check-update:
```

Comment:
wait how is that related to the File Transfer?

Reply:
I simply moved it to have it in alphabetical order